linuxacademy_dl/downloader.py

```
@@ -72,7 +72,11 @@ def __call__(self, playlist_url, save_to, file_name):
             logger.info("Skipping already existing file {}".format(final_path))
         else:
             logger.info("Downloading {}".format(final_path))
-            self.ffmpeg_process(playlist_url, final_path)
+            try:
+                self.ffmpeg_process(playlist_url, final_path)
+            except:
+                print("ffmpeg not found. Please install ffmpeg.)
```

It would be better to raise an exception, say 'DLEngineException' here..!